### ISSUE TITLE
Add option for change the font color in transparent widget mode

### ISSUE THREAD

**Issue by Alexanderlll** [martykan/forecastie#355]
Hi guys, it would be great if the color of the font can change in transparent widget mode, and be really better if the clock and the other elements can be changed independently, thanks a lot for your time

**Comment by robinpaulson** [comment]
Hmm, this sounds useful, but it's yet more things on an already crowded settings screen.

A question: in what circumstances does it need changing from the default?

**Comment by AcroBanwagon** [comment]
May be a bit late, but a useful circumstance would be when you have a light background. At least being able to change the color to black would be nice.

**Comment by robinpaulson** [comment]
Not late at all. A good answer, worth considering. I think this is a big, not an enhancement, it's a reasonably significant accessibility issue.

**Comment by robinpaulson** [comment]
I've been contemplating whether we can automatically change the widget text colour to whatever contrasts with the background. I think it's possible, but it would require knowing where on the screen the widget is placed. I assume that's possible, but neither what permissions it would require, nor how to do it. It would also be more complicated than another settings option. It's elegance does appeal though.

Alternatively, we could make the background to the widget opaque/translucent.

**Comment by AcroBanwagon** [comment]
I can't say whether or not your first idea is possible, but I think it would be better and maybe easier to either:

1. Add an option to simply toggle the text color between black and white. (or maybe let the user pick a color)
2. Make the widget opaque so the text color doesn't need to change. Maybe the user could chose an opacity level.

**Comment by robinpaulson** [comment]
I agree, I guess I was going through a thought process more than proposing a direction. I think 2. is the best, but I'm sure we'll get requests to make it configurable.

**Comment by robinpaulson** [comment]
I think I've gone round in a circle. The user specifically referenced the translucent widget, so making it opaque is not an option.....
Sigh, I'm not concentrating

**Comment by robinpaulson** [comment]
So, we're left with letting the user choose black or white for the text, option 1.

**Comment by AcroBanwagon** [comment]
That would be my preferred solution, but if you wanted to go with 2, you could make the level of opacity configurable(not just completely transparent) by adding a slider or a box to enter an opacity level.

**Comment by robinpaulson** [comment]
That sounds like more work than necessary. In either case, I probably won't refuse a pull request, but black/white is more likely to happen.
